pkg/ratelimit/runner.go

```diff
@@ -109,6 +111,9 @@ func (s *ConcurrentRunner) Start() {
 				if len(s.pendingTasks) > 0 {
 					maxDuration = time.Since(s.pendingTasks[0].submittedAt)
 				}
+				for name, cnt := range s.pendingTaskCount {
```

Comment:
Does it run too frequently?

Reply:
5 secs / op

Reply:
The metrics are collected every 15s.

Reply:
Yes, The interval should be smaller than the collected interval.